include/effects/strip/fireeffect.h

```diff
@@ -283,8 +300,9 @@ class MusicalPaletteFire : public PaletteFlameEffect, protected BeatEffectBase
                        int sparkHeight = 3,
                        bool reversed = false,
                        bool mirrored = false)
-        : PaletteFlameEffect(strName, palette, ledCount, cellsPerLED, cooling, sparking, sparks, sparkHeight, reversed, mirrored),
-          BeatEffectBase(1.00, 0.01)
+        : PaletteFlameEffect(strName, palette, _ignoreGlobalColor, ledCount, cellsPerLED, cooling, sparking, sparks, sparkHeight, reversed, mirrored),
+          BeatEffectBase(1.00, 0.01),
+          _ignoreGlobalColor(ignoreGlobalColor)
 

```

Comment:
It just occurred to me that `MusicPaletteFire` subclasses `PaletteFlameEffect`, which also holds and persists `ignorelGlobalColor`. If you pass `ignoreGlobalColor` to the `PaletteFlameEffect` constructor (as you do), then there's no need to put it in a variable or persist it to JSON in `MusicPaletteFire` as well.

Reply:
Thanks for the suggestion. Still I am still learning a lot, I had no idea how much to explicitly declare in the subclass.

Reply:
General observation: While I like the ability to converse about specific code sections, it is real hard to keep track of the conversation. So, forgive me for being redundant at times and not missing important things that are pointed out from time to time.